docker/nginx/sites-available/runestone.template

```diff
@@ -61,13 +61,17 @@ server {
         # `proxy_pass <https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass>`_: set the protocol and address of a proxied server and an optional URI to which a location should be mapped. **Tricky**: Specifying the ``location`` above with a trailing slash and including a trailing slash at the end of ``proxy_pass`` causes nginx to strip of the ``/ns`` prefix when sending it to gunicorn. Quoting the docs: "If the ``proxy_pass`` directive is specified with a URI, then when a request is passed to the server, the part of a normalized request URI matching the location is replaced by a URI specified in the directive."
         #
         # _`gunicorn socket`: This matches the ``--bind`` parameter when `gunicorn is run <run_bookserver>`.
-        proxy_pass http://unix:/run/gunicorn.sock:/;
+        proxy_pass http://unix:/run/fastapi.sock:/;
     }
 
     location / {
-        # Per `uwsgi_param <https://nginx.org/en/docs/http/ngx_http_uwsgi_module.html#uwsgi_param>`_, this passes standard CGI environment variables.
```

Comment:
I assume this is for web2py?  This is where the /runestone/ part of the URL actually helps and makes things clearer.

Reply:
Yes, only `/ns` routes to the BookServer; everything else goes to web2py.